week1/homework/src/server.js

```diff
@@ -2,14 +2,35 @@
 
 const http = require('http');
 
-/* `createServer` MUST return an instance of `http.Server` otherwise the tests
- * will fail.
- */
 function createServer(port) {
   let state = 10;
 
   const server = http.createServer((request, response) => {
-    // TODO: Write your homework code here
+    let serverResponseObject = {};
```

Comment:
You are never re-assigning `serverResponseObject` to a different value. It always continues being the same object. You are then changing the properties of the object but not changing the variable. For this reason it's better to use `const` instead of 'let'